test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -634,22 +738,24 @@ public void close() throws Exception
             int cmsNode = Iterables.getFirst(cmsNodesUp, null);
             try
             {
-                epochHistory = cluster.get(cmsNode).callOnInstance(() -> {
-                    LogState all = ClusterMetadataService.instance()
-                                                         .processor()
-                                                         .getLogState(Epoch.EMPTY, Epoch.create(Long.MAX_VALUE), false,
-                                                                      Retry.Deadline.retryIndefinitely(DatabaseDescriptor.getCmsAwaitTimeout().to(NANOSECONDS),
-                                                                                                       TCMMetrics.instance.commitRetries));
-                    StringBuilder sb = new StringBuilder("Epochs:");
-                    for (Entry e : all.entries)
-                        sb.append("\n\t\t").append(e.epoch.getEpoch()).append(": ").append(e.transform);
-                    return sb.toString();
-                });
+                SimpleQueryResult qr = Retry.retryWithBackoffBlocking(5, () -> cluster.get(cmsNode).executeInternalWithResult("SELECT epoch, kind, transformation FROM system_views.cluster_metadata_log"));
```

Comment:
rather than use class loader magic to fetch internal APIs, use the APIs we expose to operators!